Feat: LogoutSuccessHandler, RedisConfig, getExpiration 구현

segye opened the pull request:
- LogoutSuccessHandler 구현
  - 로그아웃시에 쿠키에 저장된 access-token, refresh-token, JSESSIONID 삭제
- RedisConfig 구현
  - Redis 사용시 필요
- getExpiration 구현
  - 토큰의 남은 만료 시간을 알 수 있도록 구현
  
  redis를 사용하려면
  application.yml에
  spring:
    data:
      redis:
         port: 6379
         host: localhost
 를 추가하시면 됩니다!

close #4

Commit:
Feat: LogoutSuccessHandler, RedisConfig, getExpiration 구현
- LogoutSuccessHandler 구현
  - 로그아웃시에 쿠키에 저장된 access-token, refresh-token, JSESSIONID 삭제
- RedisConfig 구현
  - Redis 사용시 필요
- getExpiration 구현
  - 토큰의 남은 만료 시간을 알 수 있도록 구현

sang218 commented on src/main/java/org/team14/webty/security/token/JwtManager.java:
```diff
@@ -1,4 +1,32 @@
+		this.secretKey = Keys.hmacShaKeyFor(secret.getBytes());
+	}
+
+	public Long getExpiration(String token) {
```
혹시 `getExpiration` -> `getExpirationTime`으로 이름 변경 가능할까요?
후에 추가할 만료 여부 확인 메소드(`isExpired`)와 헷갈릴 것 같아서요

sang218 reviewed (commented):
redis 관련 내용을 `resources/docs/application.md`에 추가해주실 수 있을까요?

DayDreaam reviewed (commented):
확인완료했습니다!

Dia218 commented:
세계님! 먼저 PR 올렸던 제 코드를 병합했더니 
세계님과 제 CookieManager랑 JwtManager가 조금 달라서 충돌이 났어요
혹시 지금 안계시면 제가 충돌 해결해서 병합해두겠습니다

segye commented:
앗 넵 감사합니다!

sang218 committed:
fix: Fix MergeConflict

- fix mergeConflict: CookieManager, JwtManager
- RedisConfig 수정: 자동 주입 문제가 발생하여 실행이 되지 않아 수정함
- application.md 수정: redis 관련 내용 추가
- LogoutSuccessHandlerTest 구현: LogoutSuccessHandler 단위 테스트
- test 파일 대규모 이동: src/main과 매칭되는 곳으로 이동

sang218 reviewed (approved):
- 충돌 해결 완료
- 오류 수정 완료
- application.md 업데이트 완료
- 단위테스트 작성 완료
